src/hb-font.cc

```diff
@@ -1168,6 +1188,24 @@ hb_font_get_glyph_from_name (hb_font_t      *font,
   return font->get_glyph_from_name (name, len, glyph);
 }
 
+/**
+ * hb_font_get_glyph_shape:
+ * @font: #hb_font_t to work upon
+ * @glyph: : The glyph ID
+ * @dfuncs:
+ * @draw_data:
+ *
+ * Fetches the glyph shape that corresponds to a glyph in the specified @font.
+ *
```

Comment:
Would be good to say that the glyph shape is 'returned' in the form of calls to the hb_draw_func_t callbacks. That is what is going to happen, right?